spec/minfraud/request_spec.rb

```diff
@@ -68,6 +68,24 @@
       expect(Minfraud).to receive(:uri).with('us_east').and_return(us_east_uri)
       Minfraud::Request.new(trans).get
     end
+
+    it 'sets a timeout on the http connection both read and open' do
+      http = double(:http).as_null_object # loose double
+      expect(Net::HTTP).to receive(:new).and_return(http)
+      expect(http).to receive(:read_timeout=).with(3)
+      expect(http).to receive(:open_timeout=).with(3)
+
+      trans = Minfraud::Transaction.new do |t|
+        t.ip = '1'
+        t.city = '2'
+        t.state = '3'
+        t.postal = '4'
+        t.country = '5'
+        t.txn_id = '6'
+        t.timeout = 3
```

Comment:
Looks like you'll eventually get a pretty obscure error:
```
NoMethodError: undefined method `zero?' for "1.5":String
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/timeout.rb:76:in `timeout'
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/net/http.rb:878:in `connect'
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/net/http.rb:863:in `do_start'
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/net/http.rb:852:in `start'
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/net/http.rb:1369:in `request'
/home/vagrant/src/minfraud-ruby/lib/minfraud/request.rb:71:in `send_get_request'
/home/vagrant/src/minfraud-ruby/lib/minfraud/request.rb:45:in `get'
/home/vagrant/src/minfraud-ruby/lib/minfraud/request.rb:51:in `get'
/home/vagrant/src/minfraud-ruby/lib/minfraud/transaction.rb:75:in `response'
/home/vagrant/src/BladeRunner/app/services/minfraud_service.rb:24:in `block in minfraud_lookup'
/usr/lib/shopify-ruby/2.1.1/lib/ruby/2.1.0/benchmark.rb:294:in `realtime'
/home/vagrant/src/BladeRunner/app/services/minfraud_service.rb:23:in `minfraud_lookup'
/home/vagrant/src/BladeRunner/app/services/minfraud_service.rb:15:in `response'
/home/vagrant/src/BladeRunner/app/filters/minfraud_filter.rb:12:in `minfraud_response'
```

I could throw an exception earlier or call the `to_f` method on the timeout passed to the gem. 

What do you think?